Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -472,6 +485,9 @@ struct PerWebProcessState {
 - (WKPageRef)_pageForTesting;
 - (NakedPtr<WebKit::WebPageProxy>)_page;
 - (RefPtr<WebKit::WebPageProxy>)_protectedPage;
+#if ENABLE(SCREEN_TIME)
+- (STWebpageController *)screenTimeWebpageController;
```

Comment:
Need a leading underscore here, because this class is public API and this method is internal.